srcpkgs/python3-pkginfo/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'python3-pkginfo'
 pkgname=python3-pkginfo
-version=1.5.0.1
-revision=4
+version=1.8.3
+revision=1
 wrksrc=pkginfo-${version}
 build_style=python3-module
-depends="python3"
 hostmakedepends="python3-setuptools"
+depends="python3"
+checkdepends="python3-wheel"
```

Comment:
instead of this, the actual test dependencies should be listed ([see here](https://bazaar.launchpad.net/~tseaver/pkginfo/trunk/view/head:/setup.py#L13))

Reply:
I added those. If I remove `python3-wheel` I get the following test error:

```
======================================================================
ERROR: test_ctor_w_dist_info (pkginfo.tests.test_installed.InstalledTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/pkginfo-1.8.3/pkginfo/tests/test_installed.py", line 101, in test_ctor_w_dist_info
    import wheel
ModuleNotFoundError: No module named 'wheel'
```

As such, I left `python3-wheel` in